Exercises/4-count-types.js

```diff
@@ -1,5 +1,31 @@
 'use strict';
 
-const countTypesInArray = null;
+const countTypesInArray = array => {
+  const counter = { };
```

Comment:
This collection contains multiple counters
```suggestion
  const counters = {};
```